bin/marcxml_to_bf2_test.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+#
+# Requires one input parameter - the path to a marcxml file.
+#
+# Convert marcxml file indicated in param to a Bibframe RDF file.
+
+BASEURI="http://ld4p-test.stanford.edu/"
+
+BF2_XSL="loc_marc2bibframe2/xsl/marc2bibframe2.xsl"
+
+marcxml_file=$1
+marcrdf_file=$(echo "${marcxml_file}" | sed s/.xml$/.rdf/)
+
+xsltproc --stringparam baseuri ${BASEURI} ${BF2_XSL} ${marcxml_file} \
+  1> ${marcrdf_file}
+
+success=$?
+if [ ${success} ]; then
+  msg="SUCCESS transformed ${marcxml_file} to ${marcrdf_file}"
+else
+  msg="FAILED to transform ${marcxml_file} to ${marcrdf_file}"
+fi
+echo "${msg}"
+
+# TODO: add an RDF/XML parser check on the output, e.g.
+# using http://librdf.org/raptor/
+# rapper -c -i rdfxml ${marcrdf_file}
```

Comment:
this is a good idea, but I would envision it as a separate step in our "pipeline" and thus it wouldn't show up in this repo.